### ISSUE TITLE
`server-transports` example does not compile

### ISSUE THREAD

**Issue by tertsdiepraam** [NLnetLabs/domain#436]
The `server-transports` example should compile with
```
cargo build --example server-transports --features="net unstable-server-transport"
```
but only compiles with 2 additional dev-dependencies and another feature flag:
```
cargo add --dev tracing_subscriber -F env-filter
cargo add --dev tokio_stream
cargo build --example server-transports --features="net unstable-server-transport unstable-client-transport"
```

**Comment by ximon18** [comment]
FYI https://github.com/NLnetLabs/domain/pull/432 fixes a similar issue in the serve-zone example.

**Comment by tertsdiepraam** [comment]
@ximon18 this was fixed by #432 as well right?

**Comment by tertsdiepraam** [comment]
Fixed by #432
